.gitignore

```diff
@@ -6,3 +6,4 @@ lib/
 npm-debug.log
 .projects.json
 yarn-error.log
+yarn.lock
```

Comment:
Can you please undo this; the yarn lock file is supposed to be checked in to git:

https://yarnpkg.com/en/docs/yarn-lock#toc-check-into-source-control

cc @Ledoux @ClemDelp